web_src/less/_review.less

```diff
@@ -16,6 +16,13 @@
   }
 }
 
+.diff-file-box .lines-code .code-inner.has-escaped::before {
+  visibility: visible;
+  content: '⚠️';
```

Comment:
Perhaps instead this icon could be a ⇄ / ⇆ or similar, so you can infer from the icon that it's bidirectional.

Reply:
I think the icon is fine, but I think a SVG would be a bit more consistent in rendering than an emoji.

Reply:
It is not just bidirectional text.